cleanlab/datalab/datalab.py

```diff
@@ -270,13 +280,24 @@ def find_issues(
                 >>> # lab.find_issues(pred_probs=pred_probs, issue_types=issue_types)
 
         """
-        issue_finder = IssueFinder(datalab=self, verbosity=self.verbosity)
+        if issue_types is not None and not issue_types:
+            warnings.warn("No issue types were specified. No issues will be found in the dataset.")
+            return None
+        no_args_passed = all(arg is None for arg in [pred_probs, features, knn_graph, issue_types])
+        if no_args_passed:
+            warnings.warn("No arguments were passed to Datalab.find_issues().")
```

Comment:
CC @elisno 

Message should clarify: 

1. what is the implication of this 
2. what should user consider doing instead

Reply:
We decided to remove this check+warning as Datalab will just not run anything useful.

As a matter of fact, some issue checks being added don't require and arguments (like the imbalanced classes issue). If it will be a default check, then this warning won't be necessary.